arch/arc/Kconfig

```diff
@@ -26,6 +26,23 @@ config CPU_ARCHS
 	  This option signifies the use of an ARC HS CPU
 
 
+choice
+	prompt "ARC Instruction Set"
+	default ISA_ARCV2
+
+config ISA_ARCV2
```

Comment:
I know that ARM using `ISA_*`, but just wondering if this is better as `ARC_ISA_V2`. This makes it easier to do `grep "config ARC"`.

Reply:
Well, canonical spelling in the "ARCvX", i.e. 'ARCv2', 'ARCv3' etc. Thus this spelling here. But I tend to agree that separated 'ARC_ISA' might be more convenient and easier to read, so why not?

Reply:
I'd prefer to stick to ISA_ARCV2 and ISA_ARCV3 - these are the regular names, and "ISA_" is commonly used as well.